src/components/avatar/index.tsx

```diff
@@ -407,7 +407,7 @@ function createStyles(props: AvatarProps) {
       lineHeight: undefined
     },
     ribbon: {
-      backgroundColor: Colors.primary,
+      backgroundColor: Colors.$backgroundPrimaryLight,
```

Comment:
I think it should be `$backgroundPrimaryHeavy`, no?

Reply:
Indeed